src/components/RouteSteps/RouteSteps.tsx

```diff
@@ -21,17 +21,21 @@ type RouteStepsProps = {
   assetSymbol?: string
   action?: StakingAction
   location: Location
+  validateAllSteps?: boolean
```

Comment:
The `validateXYZ` terminology is confusing. This isn't about validating anything, but rather about a step in the breadcrumb being done (in this case, the broadcast being successful, but I'd rather it being better abstracted than just being about the broadcast success)